ocs_ci/ocs/ui/page_objects/page_navigator.py

```diff
@@ -338,14 +338,26 @@ def navigate_block_pool_page(self):
         Navigate to block pools page
 
         """
-        logger.info("Navigate to block pools page")
+        logger.info("Navigate to Block Pools Page")
         storage_system_details = (
             self.nav_odf_default_page()
             .nav_storage_systems_tab()
             .nav_storagecluster_storagesystem_details()
         )
         storage_system_details.nav_ceph_blockpool()
-        logger.info("Now at Block pool page")
+        logger.info("Now at Block Pools page")
+
+    def navigate_client_page(self):
+        """
+        Navigate to clients page (for Provider/Client setup)
```

Comment:
This method should return instance of the class StorageClients, similarly to other navigate methods